audio/libmad/files/remove-optimizations.patch

```diff
@@ -0,0 +1,26 @@
+--- configure.ac.orig	2004-01-23 04:41:32.000000000 -0500
++++ configure.ac	2020-12-06 10:22:19.000000000 -0500
+@@ -140,18 +140,18 @@
+     case "$optimize" in
+ 	-O|"-O "*)
+ 	    optimize="-O"
+-	    optimize="$optimize -fforce-mem"
++	    : #x optimize="$optimize -fforce-mem"
```

Comment:
Don't contribute a patch that comments out code. Instead, contribute a patch that removes code. That makes the patch smaller and easier to read. Also provide a URL of the upstream bug report. If there isn't one yet, file one. We don't want to maintain patches forever; we want them included in the next upstream version of the software.

Reply:
The build setup here is very antiquated, and it's not clear that this software is even maintained anymore. I can modify the patch to remove these lines instead, but I'm not certain that this will ever be made available upstream (last release was in 2004).

Reply:
> I can modify the patch to remove these lines instead,

Please.

> but I'm not certain that this will ever be made available upstream (last release was in 2004).

Agreed.